infra-templates/vxlan/9/README.md

```diff
@@ -0,0 +1,11 @@
+## VXLAN Networking
+
+Rancher networking plugin using VXLAN overlay.
+
+### Open Ports
+
+Traffic to and from hosts requires UDP port `4789` to be open.
+
+### Upgrade options
```

Comment:
Would we want to give out options of deleting the infra service and starting a new vxlan service? would that work for people who want to keep the same environment?